xrviz/control.py

```diff
@@ -30,6 +31,7 @@ class Control(SigSlot):
     def __init__(self, data):
         super().__init__()
         self.data = data
+        self.has_cartopy = cartopy_geoviews_installed()
```

Comment:
Why the need for the attribute? Can't you refer to the global variable everywhere?